### ISSUE TITLE
Does "Polymer" here mean "polymer.js" or "LitElement"?

### ISSUE THREAD

**Issue by Westbrook** [GoogleChromeLabs/perf-track#20]
Due to some lumpy branding by the Polymer _Team_ it is often unclear in the broader community whether references to "Polymer" mean `polymer.js` (or the "Polymer Library", which is currently in [maintenance mode](https://polymer-library.polymer-project.org/) or whether they means ["LitElement"](https://lit-element.polymer-project.org/) (the spiritual successor of the Polymer Library that some may call Polymer 4). It would be good to clarify in your documentation which you are displaying this data for as it is generally understood the LitElement will generally result in better scores than the Polymer Library did due to advances in library size and renderer performance.

_If_ it isn't LitElement that is being referenced here, might I suggest that it be included as well as or in place of your current listing? Both libraries place trackable data points into the global scope that can be followed by Webappalyzer to ensure data collection, and visitors will be more appropriately educated as to the _current_ technical options they have when entering into building a front-end application.

**Comment by jimmont** [comment]
Yes LitElement should be added and included as a separate entry.

LitElement and Polymer are 2 different things. The Polymer project is a project which made several things. Many of the features in the original Polymer project were implemented natively. As a result the project's original aims are irrelevant today. Two separate libraries build on top of these newer native APIs for templates (lit-html) and the other uses this templating library and other native features collectively called Web Components providing a higher level API for what seems to broadly regarded as "reactive" UI features--this is LitElement.

Polymer's last and as I understand final release is 3.x from mid 2019, see https://en.wikipedia.org/wiki/Polymer_(library) also note the main prominent entry at the project's home: https://polymer-library.polymer-project.org/ states: "For new projects, we recommend starting with our next-generation web components library, LitElement. LitElement is a smaller, lighter successor to the Polymer library. For existing Polymer apps, we recommend upgrading to version 3.0 of the Polymer library."

**Comment by housseindjirdeh** [comment]
Sorry for the delay folks. The data currently is specific to `polymer.js` and doesn't yet include LitElement. Totally agree that it needs to be included.

We have another issue open for adding LitElement (#23) and the good news is I should be able to in a few weeks once the December CrUX run is complete 👍
